data/json/snippets/snippets.json

```
@@ -84,7 +84,7 @@
       "\"A man in a black robe came up to me, said he wanted to make a deal...\"",
       "\"Cha-cha-cha-chia!  Saw a woman today, with fungus coming out her head like hair tendrils.\"",
       "\"If only I had had more time with that Autodoc I could have saved them!  But with that damn chip in their brain those poor cyborg wil never be human again...\"",
-      "\"Save the cyborg!  Please whoever read this you must help them!  Knock them out or inactivate them I don't care!  But bring them on that Autodoc and remove the chip messing up their brain.\"",
+      "\"Save the cyborgs! Please, whoever reads this, you must help them! Knock them out or deactivate them, I don't care how you do it. But put their unconscious body on that Autodoc and remove the chip that is messing up their brain!\"",
```

Double spaces are required after sentences:
```suggestion
      "\"Save the cyborgs! Please, whoever reads this, you must help them!  Knock them out or deactivate them, I don't care how you do it.  But put their unconscious body on that Autodoc and remove the chip that is messing up their brain!\"",
```